src/applications/disability-benefits/all-claims/pages/militaryHistory.js

```diff
@@ -58,7 +58,7 @@ export const uiSchema = {
       'ui:field': ArrayField,
       'ui:options': {
         itemName: 'Service Period',
-        itemKeyForAriaLabel: 'serviceBranch',
+        itemAriaLabel: data => data.serviceBranch,
```

Comment:
Not sure how big a deal this is or how common, but if service branches are the same in two separate service periods, this would create identical aria labels. Would it be good to add some type of unique identifier per label? 
<img width="1188" alt="Screen Shot 2021-02-19 at 4 10 07 PM" src="https://user-images.githubusercontent.com/15097156/108562022-37811280-72cd-11eb-8677-943c35fe29b0.png">

Reply:
Yeah, I could include the start date:

```js
itemAriaLabel: data => `${data.serviceBranch} starting on ${data.dateRange.from}`
```

Reply:
Hmmm... so I included the date range, but I also have that same label used in the edit title.. not sure which is better "Editing service period" or this longer version

![Screen Shot 2021-02-22 at 9 37 57 AM](https://user-images.githubusercontent.com/136959/108732530-46e1a500-74f3-11eb-9b7b-162afb3cc264.png)

Reply:
Yeah I see what you mean, that is quite verbose. Would it be sufficient to shorten it to what you had suggested above?
``` 
itemAriaLabel: data => `${data.serviceBranch} starting on ${formatDate(data.dateRange.from)} 
```

Reply:
Totally your call though. Feel like this is becoming a content / wording discussion, and I don't want it to be a blocker from merging the PR 🤷

Reply:
LOL, I might just go back to "Editing service branch", but I'll see what other reviewers say, since this will be a global change.

Reply:
I also thought about passing all array data to the new function, to determine if there are duplicates, and if there are get the index and add "first", "second", etc - using https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/utilities/data/numberToWords.js